builtin/game/chat.lua

```diff
@@ -1039,6 +1039,24 @@ core.register_chatcommand("clearobjects", {
 	end,
 })
 
+core.register_chatcommand("upgrade-mapblocks", {
```

Comment:
"upgrade_mapblocks" or "upgrademapblocks" for consistency with other chat commands.